xbmc/LangInfo.cpp

```diff
@@ -667,6 +681,24 @@ LanguageResourcePtr CLangInfo::GetLanguageAddon(const std::string& locale /* = "
   return NULL;
 }
 
+bool CLangInfo::ConvertEnglishNameToAddonLocale(const std::string& langName, std::string& locale)
```

Comment:
Why not return the converted locale if found else empty string instead of this book return code approach + string refs? Would make the API a bit more sane. The cost of returning a copy is not significant in my opinion.

Reply:
so i can do it for public methods, instead for private/protected i keep ref way
existing methods use ref way and this result in a more clean code, otherwise all code become more verbose due to have to check empty string after every single method call

Reply:
yeah, fine doing it for public methods only.

Reply:
done